modules/standard/BigInteger.chpl

```diff
@@ -2894,7 +2894,18 @@ When ``n/d`` does not produce an integer, this method may produce incorrect resu
     }
   }
 
-  proc bigint.perfect_power_p() : int {
+  /*
+    .. warning::
+
+       bigint.perfect_power_p is deprecated, use bigint.isPerfectPower instead
+  */
+  deprecated
+  "bigint.perfect_power_p is deprecated, use bigint.isPerfectPower instead"
+  proc bigint.perfect_power_p() : bool {
```

Comment:
Ditto comment in your other PR about not changing the return type of the deprecated function